src/installation/index.md

```diff
@@ -1,117 +1,127 @@
+[rust-lang-book--platform-support-tier2]: https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-2
+[wiki-riscv-standard-extensions]: https://en.wikichip.org/wiki/risc-v/standard_extensions
+[rust-lang-book--platform-support-tier3]: https://doc.rust-lang.org/nightly/rustc/platform-support.html#tier-3
 
-Once that `espup` is installed, you can simply run:
-```sh
-espup install
-```
 
-This will install all the necessary tools to develop Rust applications for all supported ESP targets.
+3. To build `std` projects, you also need to install
+    - [`LLVM`][llvm-website] compiler infrastructure
+    - Other [`std` developemnt requirements][rust-esp-book-std-requirements]
+    - In your project's file `.cargo/config.toml`, add the unstable Cargo [feature][cargo-book-unstable-features] `-Z build-std`. Our [template projects][rust-esp-book-write-app-generate-project] that are discussed later in this book already have it added.
+
+
+[llvm-website]: https://llvm.org/
+[rust-esp-book-std-requirements]: #std-development-requirements
+[cargo-book-unstable-features]: https://doc.rust-lang.org/cargo/reference/unstable.html
+[rust-esp-book-write-app-generate-project]: ../writing-your-own-application/generate-project-from-template.md
+
+
+Now you should be able to build and run projects on the Espressif's RISC-V chips.
+
 
-`espup` will create an export file, by default `$HOME/export-esp.sh` on Unix systems and `%USERPROFILE%\export-esp.ps1` on Windows. This file contains some environment variables required to build projects. Make sure to source this file in every terminal before building any application:
+## Xtensa targets
+
+[espup][espup-github] is a tool that simplifies installing and maintaining the components required to develop Rust applications for the `Xtensa` architecture.
+
+[espup-github]: https://github.com/esp-rs/espup
 
+To install `espup`, run
 ```sh
-# Unix
-. $HOME/export-esp.sh
-# Windows
-%USERPROFILE%\export-esp.ps1
+cargo install espup
 ```
 
-[esp-rs/espup]: https://github.com/esp-rs/espup
-[see Usage section of the `espup` Readme]: https://github.com/esp-rs/espup#usage
+You can also directly download pre-compiled [release binaries] or use [cargo-binstall].
+
 [release binaries]: https://github.com/esp-rs/espup/releases
 [cargo-binstall]: https://github.com/cargo-bins/cargo-binstall
 
+Once `espup` is installed, do the following
+
+1. Install all the necessary tools to develop Rust applications for all supported ESP targets by running
+    ```sh
+    espup install
+    ```
+
+    `espup` will create an export file that contains some environment variables required to build projects:
+
+    - Unix systems - `$HOME/export-esp.sh`
+    - Windows - `%USERPROFILE%\export-esp.ps1`
+
+2. Make sure to source this file in every terminal before building any application:
+
+    - Unix systems, run `. $HOME/export-esp.sh`
+    - Windows, run `%USERPROFILE%\export-esp.ps1`
```

Comment:
In Windows, we create the export file, but variables are automatically injected in the user environment, and do not need to be sourced, see https://github.com/esp-rs/espup#install for more details

Reply:
Done!